d2l-typography.scss

```diff
@@ -160,3 +160,16 @@
 		line-height: 1.2rem;
 	}
 }
+
+@mixin d2l-label-text() {
```

Comment:
Same goes for the rem units down here and in the responsive section. Just take what you have, divide it over 20 and put `rem` instead of `px`.

Reply:
Done.